src/Plugin/Field/FieldWidget/EnhancedButtonLinkWidget.php

```diff
@@ -57,6 +58,7 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
       '#default_value' => isset($items[$delta]->options['size']) ? $items[$delta]->options['size'] : '',
       '#description' => $this->t('Select the size of the button.'),
       '#options' => [
+        'default' => $this->t('default'),
```

Comment:
Please update 'default' to 'Default', so it should be same as:
`$this->t('Default'),`
+ do the same for title of 'normal', 'small' and 'big'